File: slack-poster/src/test/java/dev/number6/slackposter/dagger/FakeRequestPort.java
package dev.number6.slackposter.dagger;

import dev.number6.slack.port.RequestPort;

public class FakeRequestPort implements RequestPort {
    @Override
    public String getMethod() {
        return null;
    }

    @Override
    public String getHeader(String headerName) {
        return null;
    }

    @Override
    public String getUrl() {
        return null;
    }
}

File: slack-reader/src/test/java/dev/number6/slackreader/dagger/FakeRequestPort.kt
package dev.number6.slackreader.dagger

import dev.number6.slack.port.RequestPort

class FakeRequestPort : RequestPort {
    override fun getMethod(): String {
        return "null"
    }

    override fun getHeader(headerName: String): String {
        return "null"
    }

    override fun getUrl(): String {
        return "null"
    }
}